uptane-standard.md

```diff
@@ -561,20 +561,22 @@ The Timestamp metadata SHALL contain the following information:
 
 Repository mapping metadata informs a primary ECU about which repositories to trust for images or image paths. Repository mapping metadata MUST be present on all primary ECUs, and MUST contain the following information:
 
-* A list of repository names and one or more URLs at which the named repository can be accessed. At a minimum, this MUST include the Director and Image repositories.
-* A list of mappings of image paths to repositories, each of which contains:
+* A list of repository names, each with one or more URLs at which the repository can be accessed.  The first repository name in this list SHALL be known as the Director repository.
+* A list of mappings of image paths to repositories.  Each mapping MUST contain:
     * A list of image paths. Image paths MAY be expressed using wildcards, or by enumerating a list, or a combination of the two.
-    * A list of repositories that MUST sign the targets metadata for the image paths.
+    * A list of repositories (at least two) that MAY sign the targets metadata for the image paths.  The first must always be the Director repository (the same repository that is listed first in the list of repository names).
```

Comment:
Is this: 'The first must always be the Director repository' a MUST?

Reply:
Also, does at least one repository have to sign the targets metadata?

Reply:
@mnm678 , @iramcdonald:
It is intended as a MUST, yes, and I should fix the casing.  While we're on this, though, one could argue that this is too particular a requirement.  That would be helpful feedback.  The requirement for the procedure written here is that there must be one repository that will be regarded as the Director, and that it is defined at the level of the mapping metadata (not individual mappings).  I could instead not define how the client knows which repo is the Director repository.  Is that too loose?  Thoughts?